docs/admin/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 8
+sidebar_label: "Kwil Admin Tool"
+id: admin
+title: Kwil Node Administration Tool
+description: The Kwil admin tool
+slug: /admin
+---
+
+The Kwil node administration tool is a command line application called
+`kwil-admin`, which provides the following functionality to a node operator:
+
+1. Quick test network configuration.
+2. Node key generation.
+3. Interaction with a running `kwild` process via the node's authenticated gRPC interface.
+4. Validator operations.
+
+```mdx-code-block
+import DocCardList from '@theme/DocCardList';
+
+<DocCardList />
+```
```

Comment:
Doing this instead of auto-generated index page via _category_.json file so we can have actual page content above it.

Reply:
I like this a lot.  We should adopt this across more of the docs.